GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/TestRailManager/Model/CaseCustomFields.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace GherkinSyncTool.Synchronizers.TestRailSynchronizer.TestRailManager.Model
+{
+    //TODO: improve to use own test case template
+    public class CaseCustomFields
+    {
+        [JsonProperty("custom_preconds")] 
+        public string CustomPreconditions { get; init; }
+        
+        [JsonProperty("custom_automation_coverage")]
+        public int CustomAutomationCoverage { get; init; } = 1;
+
+        [JsonProperty("custom_steps_separated")]
+        public List<CustomStepsSeparated> CustomStepsSeparated { get; init; }
+        
+        [JsonProperty("custom_steps")] 
+        public string CustomSteps { get; init; }
```

Comment:
Do we really need "CustomSteps" when there is a property CustomStepsSeparated?

Reply:
Yes, this is temporary, because of the bug in TestRail client lib.